Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -76,8 +77,7 @@ class ContentBlockerHelper {
     init(tab: Tab, profile: Profile) {
         self.ruleStore = WKContentRuleListStore.default()
         if self.ruleStore == nil {
-            print("WKContentRuleListStore unavailable.")
-            assert(false)
+            assert(false, "WKContentRuleListStore unavailable.")
```

Comment:
On further thought the code no-ops anywhere the ruleList is nil, should I just do the assert() as you suggested ?